node_app/client/views/generate/index.ejs

```diff
@@ -1,70 +1,91 @@
 <% layout('layouts/boilerplate.ejs') %>
 
-<%- include("../partials/background_animation.ejs") %>
+    <%- include("../partials/background_animation.ejs") %>
 
-<div class="canvas">
+        <div class="canvas">
 
-    <!-- Call to action -->
-    <div id="callToAction">
-        <h2>Generate your own track with Neutral Networks</h2>
-    </div>
+            <!-- Call to action -->
+            <div id="callToAction">
+                <h1>Generate your own track with Neutral Networks</h1>
+                <h5>This is a Noise Enricher, a neural net that generates music, including rudimentary singing, as raw
+                    audio </h5>
+            </div>
 
-    <!-- Form module -->
-    <form action="/generate" id="generateTrack" method="POST" enctype="multipart/form-data">
-        <div id="uploadTrack">
-            <h5>Upload your track</h5>
+            <!-- Form module -->
+            <form action="/generate" id="generateTrack" method="POST" enctype="multipart/form-data">
+                <div id="uploadTrack">
+                    <h5>Upload your track</h5>
+                    <!--
             <div id="uploadFileInput">
                 <input type="file" name="audio" class="form-control">
             </div>
-            <br>
-        </div>
-
-        <!-- 3 gray blocks -->
-        <div id="allParams">
-            <section id="styleSection">
-                <p>Choose a style</p>
-                <div id="styleSelection">
-                    <div>
-                        <label for="rockRange" class="form-label">Rock</label>
-                        <input name="rockRange" type="range" class="form-range" min="0" max="6" id="rockRange">
-                    </div>
-                    <div>
-                        <label for="tranceRange" class="form-label">Trance</label>
-                        <input name="tranceRange" type="range" class="form-range" min="0" max="6" id="tranceRange">
-                    </div>
-                    <div>
-                        <label for="classicRange" class="form-label">Classic</label>
-                        <input name="classicRange" type="range" class="form-range" min="0" max="6"
-                               id="classicRange">
+            -->
+                    <div id="uploadFileInput">
+                        <span class="input-group-text btn btn-primary" id="inputGroup-sizing-default">Choose a
+                            file</span>
+                        <input type="text" name="audio" class="form-control" aria-label="Sizing example input"
```

Comment:
@giannamoth input type should be `type="file"`, it won't work otherwise, unfortunately :(

Reply:
hehe I was imagine so 😄  No worries, I'm gonna set it back to what it was 😎